stable/mlrun/templates/api-chief-deployment.yaml

```diff
@@ -180,9 +180,13 @@ spec:
           ports:
           - name: grpc
             containerPort: {{ .Values.api.sidecars.logCollector.listenPort }}
-          {{- if .Values.api.sidecars.logCollector.livenessProbe }}
           env:
-          {{- end }}
+          - name: MLRUN_LOG_COLLECTOR__NAMESPACE
+            valueFrom:
```

Comment:
we already have convention for namespace, I think we might want this to be aligned.
In mlrun its called `MLRUN_NAMESPACE`